README.md

```diff
@@ -1,9 +1,9 @@
 # chapel-vscode extension
 
-## Features
+## Core Features
 
 - Intellisense
-  - GotoDefinition: jump to a symbols definition
+  - GotoDefinition: jump to a symbol's definition
   - Hover: hover over a symbol to view its signature and documentation
   - Code Completion: suggest completion for symbols in the current file
   - Symbol List: see all symbols for a file
```

Comment:
```suggestion
  - Symbol List: see all symbols in a file
```
Not sure if "for a file" means something different than what I understood, but "in" seems to be the better wording here.